src/main/java/com/minecolonies/coremod/event/EventHandler.java

```diff
@@ -785,11 +675,14 @@ public static void onWorldLoad(@NotNull final WorldEvent.Load event)
     @SubscribeEvent
     public static void onWorldUnload(@NotNull final WorldEvent.Unload event)
     {
-        Log.getLogger().warn("World unload");
-
         if (event.getWorld() instanceof World)
         {
             IColonyManager.getInstance().onWorldUnload((World) event.getWorld());
         }
+        if (event.getWorld().isRemote() && event.getWorld().getDimension().getType() == DimensionType.OVERWORLD)
```

Comment:
Why only for overworld?

Reply:
its for removing the colony views on client side once you leave a world, and thats when the overworld is unloaded

Reply:
ok